src/app/tests/suites/certification/Test_TC_S_2_3.yaml

```diff
@@ -459,7 +530,214 @@ tests:
                 value: "y"
 
     - label:
-          "Step 6a: TH sends a CopyScene command to DUT with the mode field set
+          "Step 6a: TH sends a AddScene command to group G1 with the GroupID
+          field set to G1, the SceneID field set to 0x01, the TransitionTime
+          field set to 1000 (1s) and a set of extension fields appropriate to
+          AC1."
+      PICS: S.S.C00.Rsp && PICS_SDK_CI_ONLY
```

Comment:
why is this CI only and the below manual?

Reply:
These are the same steps. The CI one assumes the presence of the level control cluster, which is present on an all-clusters-app. The step below lets the tester decide what to use for an appropriate AC for the DUT.